db/config.go

```diff
@@ -30,7 +29,7 @@ func DB() *Database {
 		db = &Database{}
 		var c *aws.Config
 		c = &aws.Config{
-			Region:   aws.String(_const.DBREIGON),
+			Region:   aws.String(config.GlobalConfig().DBRegion),
```

Comment:
Question, why do we pull the GlobalConfig object here, but in the `Endpoint` below we use the `con` instance to pull it? Wouldn't it be better to keep it consistent?

Reply:
Oops! Good Catch ;)